web/pdf_layer_viewer.js

```diff
@@ -94,7 +94,7 @@ class PDFLayerViewer extends BaseTreeViewer {
       element.textContent = this._normalizeTextContent(name);
       return;
     }
-    element.textContent = await this.l10n.get("additional_layers");
+    element.textContent = await this.l10n.get("additional_-layers");
```

Comment:
This looks like a typo, looking at the `.ftl` file I'm assuming that you meant:
```suggestion
    element.textContent = await this.l10n.get("additional-layers");
```